.gitignore

```diff
@@ -1,3 +1,5 @@
 target
 .sbtopts
 project/.sbt
+.idea/
```

Comment:
Please remove these lines, it's developer's preference not project related

Reply:
@ktonga sure thing. Done!